src/lib.rs

```diff
@@ -33,8 +33,9 @@ extern crate hyper;
 extern crate "typemap" as tmap;
 extern crate plugin;
 extern crate "modifier" as modfier;
-extern crate error;
+extern crate "error" as err;
```

Comment:
Maybe it's better to rename `error` to something else? I usually rename it too in my own code  [same thing](https://github.com/rustless/valico/commit/e40f668b893aee50e4ba18759672c373f380e735?diff=unified#diff-dd5f1b0c97a16f836553c88faac50f8aR3).